user-org-util/src/main/java/org/sunbird/util/ProjectLogger.java

```diff
@@ -164,4 +164,54 @@ private static String getBELog(
     }
     return jsonMessage;
   }
+  /**
+   * This method will be called from Application class(default startup class). which will
+   * automatically set the Log level by taking its value from system environment file.
+   */
+  public static void setUserOrgServiceProjectLogger() {
```

Comment:
setUserOrgServiceProjectLogger method need to take loglevel as input.  getLevelFromEnv method should be called before that and value need to be passed here.